multimodal/src/autogluon/multimodal/optimization/RKDLoss.py

```diff
@@ -0,0 +1,70 @@
+# Reference: https://github.com/lenscloth/RKD/blob/master/metric/loss.py
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from .utils import pdist
+from typing import Optional
+
+
+class RKDDistance(nn.Module):
```

Comment:
Do we have to make the distance function a `nn.Module`? How about just a normal function?

Reply:
I noticed that in many repositories loss functions are implemented as nn.Module. I write them together into a RKDLoss class to simplify the codes.